assets/stylesheets/bootstrap/_forms.scss

```diff
@@ -143,7 +143,6 @@ output {
   // HTML5 says that controls under a fieldset > legend:first-child won't be
   // disabled if the fieldset is disabled. Due to implementation difficulty, we
   // don't honor that edge case; we style them as disabled anyway.
-  &[disabled],
   &[readonly],
```

Comment:
In the example I am using this for, you tick a check box and it copies values from one form to the other. The copied to location becomes disabled until you un-check the box. Would `readonly` still be suitable here?